src/library.js

```diff
@@ -1398,10 +1398,14 @@ LibraryManager.library = {
   llvm_sqrt_f64: 'Math_sqrt',
   llvm_pow_f32: 'Math_pow',
   llvm_pow_f64: 'Math_pow',
+  llvm_powi_f32: 'Math_pow',
+  llvm_powi_f64: 'Math_pow',
   llvm_log_f32: 'Math_log',
   llvm_log_f64: 'Math_log',
   llvm_exp_f32: 'Math_exp',
   llvm_exp_f64: 'Math_exp',
+  llvm_trunc_f64: 'Math_trunc',
```

Comment:
Are there such things as `llvm_trunc_f32` and `llvm_floor_f32`? If so, could add those as well?